tests/Unit/Evolution/Systems/ScalarWave/Test_UpwindPenaltyCorrection.cpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <algorithm>
+#include <array>
+#include <cstddef>
+#include <memory>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/ScalarWave/UpwindPenaltyCorrection.hpp"
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/TestHelpers.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeArray.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+template <size_t Dim>
+void penalty_flux(
+    const gsl::not_null<Scalar<DataVector>*> boundary_correction_pi,
+    const gsl::not_null<tnsr::i<DataVector, Dim, Frame::Inertial>*>
+        boundary_correction_phi,
+    const gsl::not_null<Scalar<DataVector>*> boundary_correction_psi,
+
+    const Scalar<DataVector>& constraint_gamma2,
+
```

Comment:
Blank lines between disjoint groups of inputs are great. Are we adopting this as a convention, or this is a one-off use?

Reply:
I view it more as a one-off case. It's helpful if you have 20 arguments like we do here, but if we only have 5 arguments, putting in 4 blank lines around them seems rather excessive. My view on it is "Does it help improve readability or just add noise?" If it improves readability my vote is to do it, if it just adds a lot of blank lines because every function argument is "distinct" then I don't think it helps